src/main/java/io/github/cgi/zabbix/api/Utils.java

```diff
@@ -1,84 +1,64 @@
 package io.github.cgi.zabbix.api;
 
-import org.codehaus.jackson.map.ObjectMapper;
-import org.codehaus.jackson.map.SerializationConfig;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 
-
 /**
  * Created by cgi on 01.03.2016.
  * Support Class for converting Objects to JSON
  */
 public class Utils {
-    static ObjectMapper mapper = new ObjectMapper();
-    static ObjectMapper mapperIndent = new ObjectMapper();
-
-
-    public static String toJson(Object o) {
-        String s = null;
-        try {
-            s = mapper.writeValueAsString(o);
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-        return s;
-
-    }
-
-    public static String toJsonWithIndent(Object o) {
-        String s = null;
-        mapperIndent.getSerializationConfig().enable(SerializationConfig.Feature.INDENT_OUTPUT);
-        try {
-            s = mapperIndent.writeValueAsString(o);
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-        return s;
-
-    }
-
-
-    public static byte[] getBytesFromInputStream(InputStream inputStream) {
+	static ObjectMapper mapper = new ObjectMapper();
 
-        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+	public static byte[] getBytesFromInputStream(InputStream inputStream) {
 
-        try{
-        byte[] buffer = new byte[1024];
-        int length;
+		ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
```

Comment:
ik zou deze outputStream in een try-with-resource block zetten om auto closing te realiseren. Anders loop je de kans dat je deze vergeet te sluiten.

Reply:
Het wel of niet sluiten van de stream heb ik niet aangepast. Volgens mij zijn deze changes voornamelijk formatting, maar het sluiten lijkt inderdaad niet gedaan te worden. Sowieso was dit een nogal omslachtige manier van het lezen van bytes uit een inputstream. Ik heb het aangepast